### PR TITLE
[scripts][common-healing] Add dead status to output

### PR DESCRIPTION
**Pull request by MahtraDR**
Allows us to detect whether our touch target is dead or not, and perhaps act accordingly.

```
2022-11-23-3.log:"[exec1: {\"wounds\"=>{}, \"parasites\"=>{}, \"poisoned\"=>false, \"diseased\"=>false, \"dead\"=>true, \"score\"=>0}]"
2022-11-23-3.log:"[exec1: {\"wounds\"=>{}, \"parasites\"=>{}, \"poisoned\"=>false, \"diseased\"=>false, \"dead\"=>false, \"score\"=>0}]"
```

In my case, I use it to determine whether I want to link unity (which will kill my empath if the person being healed is a corpse...) or take all quick, which won't kill my empath ever.

### CODE REVIEW

File: common-healing.lic [hunk 1]
```diff
@@ -138,17 +138,24 @@ module DRCH
       /^[\w]+ (?:body|skin) is covered (?:in|with) open oozing sores/
     ])
 
+    dead_regex = Regexp.union([
+      /^(He|She) is dead/
+    ])
+
```
Comment (asechrest) [inline common-healing.lic:144]: What is the purpose of `Regexp.union` here, given you're only specifying one regex pattern? `.union` will give a union of multiple patterns.

Comment (MahtraDR) [reply common-healing.lic:144]: In an earlier draft I had the two lines separate and followed the existing convention. I don't think we'll ever had to add a line here though. I'll fix in a bit.

Comment (MahtraDR) [reply common-healing.lic:144]: Actually, I rather keep it like this. It's all the same for the result, but this syntax leaves open simple additions for this regex (even if I can't think of any today) and is simple to read.

**Comment by Dartellum** [comment]
This update for when using `empathylink`?

**Comment by MahtraDR** [comment]
> This update for when using `empathylink`?

I hadn't considered that use case, actually. Should work too. 

I use it for random healing with an alias. If they're dead I `take all quick` if they are alive, I `link unity`.

**Pull request merged**